src/main/java/roomescape/domain/member/service/MemberService.java

```diff
@@ -0,0 +1,38 @@
+package roomescape.domain.member.service;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import roomescape.domain.member.domain.Member;
+import roomescape.domain.member.repository.MemberRepository;
+import roomescape.global.exception.ClientIllegalArgumentException;
+
+@Service
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    public MemberService(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    public List<Member> findAll() {
+        return memberRepository.findAll();
+    }
+
+    public Member findMemberById(Long id) {
+        Optional<Member> member = memberRepository.findById(id);
+        if (member.isEmpty()) {
+            throw new ClientIllegalArgumentException("없는 member를 조회 했습니다.");
+        }
+        return member.get();
```

Comment:
간단하게 줄일 수는 없을까요?

Reply:
```suggestion
        Optional<Member> member = memberRepository.findById(id);
        return member.orElseThrow(() -> new ClientIllegalArgumentException("없는 member를 조회 했습니다."));
```
이렇게 할 수 있습니다! 말해주신 것 포함해서 비슷한 구조 모두 orElseTrow()로 수정했습니다!